drivers/pinctrl/pinctrl_npcx.c

```diff
@@ -23,6 +24,16 @@ struct npcx_pinctrl_config {
 
 const pinctrl_soc_pin_t io_pad_conf[] = NPCX_IO_PAD_CONF_LIST;
 
+/* Collect all peripherals which need to set pinmux back to GPIO */
+#define NPCX_PERIPH_MUX_ITEMS(node_id)					\
+	COND_CODE_1(DT_PROP(node_id, pinmux_gpio),			\
+		(Z_PINCTRL_NPCX_PERIPH_PINMUX_INIT(node_id, pinmux)), ())
+
+#define NPCX_PERIPH_MUX_LIST						\
+	{DT_FOREACH_CHILD(DT_NODELABEL(pinctrl), NPCX_PERIPH_MUX_ITEMS)}
+
+const pinctrl_soc_pin_t periph_muxes[] = NPCX_PERIPH_MUX_LIST;
```

Comment:
Is this meant to be temporary?  This duplicates data from the individual drivers, correct?

Reply:
Done.